buildSrc/src/main/java/org/springframework/boot/build/ConventionsPlugin.java

```diff
@@ -127,6 +127,7 @@ private void applyJavaConventions(Project project) {
 			});
 			project.getTasks().withType(Jar.class, (jar) -> {
 				project.afterEvaluate((evaluated) -> {
+					jar.metaInf((metaInf) -> metaInf.from(project.getRootProject().file("LICENSE.txt")));
```

Comment:
I am not sure if we want to ship our license or something a bit more complete as the one [Spring Framework use](https://github.com/spring-projects/spring-framework/blob/master/src/docs/dist/license.txt).

Reply:
Let me know if I can help with something. Seems to be outside of my control at the moment, though!?

Reply:
I don't think the Framework one matches our needs as it includes information about CGLib and ASM which Framework jarjars. The file in the root of the repo looks like a better fit to me for most of our modules at least. We may need to do something different for `spring-boot-configuration-processor` as it shades some JSON stuff.

Reply:
> I don't think the Framework one matches our needs as it includes information about CGLib and ASM which Framework jarjars. 

I didn't literally mean that file but more something with more details than just our license. I was actually referring [to the bottom of the file](https://github.com/spring-projects/spring-boot/issues/20024#issuecomment-585151961). Not feeling strongly about it though.